project/YaoKuang/README.rst

```diff
@@ -0,0 +1,87 @@
+SimpleArray for Multi-Dimensional Field
+
+(Modmesh) SimpleArray: a simple C++ library for multi-dimensional arrays that provide seamless interface with numpy ndarray. Download this file.
```

Comment:
Don't copy and paste.  You can follow the example topic, but should write your own proposal.